ffi/capi/bindings/c/Bidi.h

```diff
@@ -18,25 +18,16 @@
 
 
 
-
```

Comment:
issue: why does this remove all blank lines between items?

Reply:
I removed them during an earlier iteration as a way to get rid of the whitespace, then decided to go about it a different way, for the overall strategy, but in the header files I actually think having them be next to each other is nice, it gives a clearer sense of sections.

Reply:
clearer sense of sections? which sections? lines 24-25 should be grouped, this changes removes that grouping